tests/integrationTests/tests/python_linehighlight/__init__.py

```diff
@@ -49,6 +49,10 @@ def initialize(test):
 
 ############################################################################
 
+@testcase
+def schema_validation(test):
+    config_path = os.path.join(test.testcase_path, 'assignment_config', 'complete_config.json')
```

Comment:
Should this test have `cleanup(test)` at the start of it like the others?

Reply:
This testcase doesn't have a cleanup function.

Reply:
I think a better question is why does most of the tests then have a cleanup function and this one doesn't? Should it have a cleanup? Should the cleanup be moved to be a pre-processor step for any functions run under the `@testcase` annotation?